src/api/index.md

```diff
@@ -2165,6 +2165,16 @@ type: api
   })
   ```
 
+  Sort using two keys:
+
+  ``` html
+  <ul>
+    <li v-for="user in users | orderBy ['lastName', 'firstName']">
```

Comment:
So turns out this makes it more complex than I thought - I'll take a stab at rewriting it to see if it can be made simpler.